client/packages/cli/src/index.js

```diff
@@ -497,16 +498,16 @@ async function handlePull(bag, opts) {
     opts,
   );
   if (!ok) return;
-  await pull(bag, appId, pkgAndAuthInfo);
+  await pull(bag, appId, pkgAndAuthInfo, opts);
```

Comment:
Similarily, I think you can skip passing an `opts` here, because this library has a global way to access the bag of passed in opts: 

Something like what is happening in `promptOk`: 

https://github.com/instantdb/instant/blob/main/client/packages/cli/src/index.js#L1316